src/PasswordResetController.js

```diff
@@ -45,6 +45,34 @@ function (Okta, FormController, FormType, ValidationUtil, FooterSignout, TextBox
     Form: {
       save: Okta.loc('password.reset', 'login'),
       title: Okta.loc('password.reset.title', 'login'),
+      subtitle: function () {
+        var policy = this.options.appState.get('policy');
+        if (!policy || !policy.complexity) {
+          return;
+        }
+
+        var complexity = policy.complexity;
+        var requirements = [];
+
+        complexity.minLength &&
+          requirements.push(Okta.loc('password.complexity.length', 'login', [complexity.minLength]));
+
+        complexity.minLowerCase && requirements.push(Okta.loc('password.complexity.lowercase', 'login'));
```

Comment:
There's a lot of duplicated logic here (plus in general I'm not a fan of using && when we really should be using an if statement).

Can we set up a map instead and iterate over the complexity requirements? Something like:
```javascript
var foo = {
  'minLength': 'password.complexity.length',
  // etc
};

var requirements = _.map(policy.complexity, function (val, type) {
  return Okta.loc(foo[type], 'login', [val]);
});

```